client/components/card-heading/style.scss

```diff
@@ -1,3 +1,6 @@
+@import "@automattic/typography/styles/variables";
+@import "../../assets/stylesheets/shared/mixins/_breakpoints";
```

Comment:
Could we use `@import "calypso/assets/stylesheets/shared/mixins/breakpoints";` instead as suggested in p4TIVU-9sM-p2? Thank you!

Reply:
Thanks for the suggestion, changed.